frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -486,6 +486,18 @@ describe("scenarios > collection_defaults", () => {
       cy.findByText("4 items selected");
     });
 
+    it("should be possible to deselect all items using checkbox (metabase#14705)", () => {
+      cy.visit("/collection/root");
+      selectItemUsingCheckbox("Orders");
+      cy.findByText("1 item selected").should("be.visible");
+      cy.icon("dash").click();
+      bulkActionsPanel().within(() => {
+        cy.icon("check").click();
+      });
+      cy.icon("check").should("not.exist");
+      bulkActionsPanel().should("not.be.visible");
+    });
+
```

Comment:
Let's merge this with the test above and keep it together. It will save us some lines and a couple of second of running time. This is also one of the [recommended good practices in Cypress](https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion).